libyul/optimiser/Suite.cpp

```diff
@@ -209,37 +211,38 @@ map<string, unique_ptr<OptimiserStep>> const& OptimiserSuite::allSteps()
 map<string, char> const& OptimiserSuite::stepNameToAbbreviationMap()
 {
 	static map<string, char> lookupTable{
-		{BlockFlattener::name,                'f'},
-		{CircularReferencesPruner::name,      'l'},
-		{CommonSubexpressionEliminator::name, 'c'},
-		{ConditionalSimplifier::name,         'C'},
-		{ConditionalUnsimplifier::name,       'U'},
-		{ControlFlowSimplifier::name,         'n'},
-		{DeadCodeEliminator::name,            'D'},
-		{EquivalentFunctionCombiner::name,    'v'},
-		{ExpressionInliner::name,             'e'},
-		{ExpressionJoiner::name,              'j'},
-		{ExpressionSimplifier::name,          's'},
-		{ExpressionSplitter::name,            'x'},
-		{ForLoopConditionIntoBody::name,      'I'},
-		{ForLoopConditionOutOfBody::name,     'O'},
-		{ForLoopInitRewriter::name,           'o'},
-		{FullInliner::name,                   'i'},
-		{FunctionGrouper::name,               'g'},
-		{FunctionHoister::name,               'h'},
-		{LiteralRematerialiser::name,         'T'},
-		{LoadResolver::name,                  'L'},
-		{LoopInvariantCodeMotion::name,       'M'},
-		{NameSimplifier::name,                'N'},
-		{ReasoningBasedSimplifier::name,      'R'},
-		{RedundantAssignEliminator::name,     'r'},
-		{Rematerialiser::name,                'm'},
-		{SSAReverser::name,                   'V'},
-		{SSATransform::name,                  'a'},
-		{StructuralSimplifier::name,          't'},
-		{UnusedFunctionParameterPruner::name, 'p'},
-		{UnusedPruner::name,                  'u'},
-		{VarDeclInitializer::name,            'd'},
+		{BlockFlattener::name,                      'f'},
```

Comment:
No whitespace-based alignment across lines is a good rule to have in the coding standard :)